chromium_src/ui/accessibility/accessibility_features.cc

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "ui/accessibility/accessibility_features.h"
+
+#define IsReadAnythingDocsIntegrationEnabled \
+  IsReadAnythingDocsIntegrationEnabled_ChromiumImpl
+
+#include "src/ui/accessibility/accessibility_features.cc"
+
+#undef IsReadAnythingDocsIntegrationEnabled
+
+namespace features {
+bool IsReadAnythingDocsIntegrationEnabled() {
```

Comment:
why aren't we just disabling the feature? I we don't want to use `ReadAnythingService` at all we should stub out the service.

Reply:
Good point, disabled here https://github.com/brave/brave-core/pull/25340/commits/a025f239cf4ff615bb09b8055d0e3dd361b8fecc